daemon/graphdriver/lcow/lcow_svm.go

```diff
@@ -0,0 +1,394 @@
+// +build windows
+
+package lcow
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"io"
+
+	"github.com/Microsoft/hcsshim"
+	"github.com/jhowardmsft/opengcs/gogcs/client"
+	"github.com/sirupsen/logrus"
+)
+
+// Code for all the service VM management for the LCOW graphdriver
+
+var errVMisTerminating = errors.New("service VM is shutting down")
+var errVMUnknown = errors.New("service vm id is unknown")
+var errVMStillHasReference = errors.New("Attemping to delete a VM that is still being used")
+
+// serviceVMMap is the struct representing the id -> service VM mapping.
+type serviceVMMap struct {
+	sync.Mutex
+	svms map[string]*serviceVMWithRef
+}
+
+// serviceVMWithRef is our internal structure representing an item in our
+// map of service VMs we are maintaining.
+type serviceVMWithRef struct {
```

Comment:
More nit than anything else, I was kind of confused to start with by the name here. Perhaps `serviceVMMapItem` would be clearer?